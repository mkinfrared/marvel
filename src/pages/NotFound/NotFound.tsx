import React from "react";

const NotFound = () => {
  return (
    <div>
      <p>404 | Page not found</p>
    </div>
  );
};

export default React.memo(NotFound);
