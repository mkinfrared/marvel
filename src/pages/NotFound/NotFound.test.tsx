import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";

import NotFound from "./NotFound";

describe("<NotFound />", () => {
  const Component = (
    <MemoryRouter>
      <NotFound />
    </MemoryRouter>
  );

  it("should be defined", () => {
    const { container } = render(Component);

    expect(container).toBeDefined();
  });

  it("match the snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });
});
