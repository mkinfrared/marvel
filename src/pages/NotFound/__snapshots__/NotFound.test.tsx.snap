// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> match the snapshot 1`] = `
<div>
  <div>
    <p>
      404 | Page not found
    </p>
  </div>
</div>
`;
